plugins/janus_videoroom.c

```diff
@@ -1671,7 +1541,47 @@ static void janus_videoroom_message_free(janus_videoroom_message *msg) {
 	g_free(msg);
 }
 
+void janus_videoroom_codecstr(janus_videoroom * videoroom, char* audio_codecs, char* video_codecs, int str_len, const char* split)
+{
+	if(audio_codecs){
+		audio_codecs[0] = 0;
+		g_snprintf(audio_codecs, str_len, "%s", janus_audiocodec_name(videoroom->acodec[0]));
+		if (videoroom->acodec[1] != JANUS_AUDIOCODEC_NONE) {
+			g_strlcat(audio_codecs, split, str_len);
+			g_strlcat(audio_codecs, janus_audiocodec_name(videoroom->acodec[1]), str_len);
+		}
+		if (videoroom->acodec[2] != JANUS_AUDIOCODEC_NONE) {
+			g_strlcat(audio_codecs, split, str_len);
+			g_strlcat(audio_codecs, janus_audiocodec_name(videoroom->acodec[2]), str_len);
+		}
+	}
+	if(video_codecs){
+		video_codecs[0] = 0;
+		g_snprintf(video_codecs, str_len, "%s", janus_videocodec_name(videoroom->vcodec[0]));
+		if (videoroom->vcodec[1] != JANUS_VIDEOCODEC_NONE) {
+			g_strlcat(video_codecs, split, str_len);
+			g_strlcat(video_codecs, janus_videocodec_name(videoroom->vcodec[1]), str_len);
+		}
+		if (videoroom->vcodec[2] != JANUS_VIDEOCODEC_NONE) {
+			g_strlcat(video_codecs, split, str_len);
+			g_strlcat(video_codecs, janus_videocodec_name(videoroom->vcodec[2]), str_len);
+		}
+	}
+}
 
+void janus_videoroom_reqfir(janus_videoroom_publisher* publisher, const char* reason){
+	/* Send a FIR to the new RTP forward publisher */
```

Comment:
If this is a generic, and shared, way of triggering PLI/FIR messages (as suggested by the `reason` above), then I guess the comment above is broken?